eg/Classes/Log.py

```diff
@@ -38,6 +38,42 @@
 INFO_ICON = eg.Icons.INFO_ICON
 ERROR_ICON = eg.Icons.ERROR_ICON
 NOTICE_ICON = eg.Icons.NOTICE_ICON
+DEBUG_ICON = eg.Icons.DEBUG_ICON
+WARNING_ICON = eg.Icons.WARNING_ICON
+
+
+def _build_notice(icon, args):
+    strs = [
+        str(eg.Tasklet.GetCurrentId()),
+        str(currentThread().getName()) + ":"
+    ]
+
+    for arg in args:
+        arg = str(arg).strip()
+        if icon == WARNING_ICON:
+            arg = arg.replace('Traceback', 'Warning')
+        strs += [arg]
+
+    msg = ' '.join(strs)
+    if icon == DEBUG_ICON:
+        msg = 'DEBUG: ' + msg.replace('\n', '\nDEBUG: ')
+
+    elif icon == WARNING_ICON:
+        msg = 'WARNING: ' + msg.replace('\n', '\nWARNING: ')
+
+    std_msg = (
+        strftime("%H:%M:%S: ") +
+        msg.replace('\n', '\n' + strftime("%H:%M:%S: "))
+    )
```

Comment:
I have to add ` + "\n"` here or in pycharm console all output will be on one line. Is it only on machine?

Reply:
I am not sure what you mean... It works fine when i run EG

Reply:
I got ya. The output into log.txt was all on one line. because i forgot to add the '\n'. I have fixed the problem